thunder/tests/test_inplace_functionalization.py

```diff
@@ -137,31 +139,52 @@ def turn_off_tf32_and_set_seed(monkeypatch):
     torch.manual_seed(42)
 
 
+@instantiate(
+    dtypes=(thunder.float32, thunder.float64),
+    devicetypes=(devices.DeviceType.CUDA,),
+    decorators=(pytest.mark.parametrize("train", (False, True)),),
+)
 @requiresCUDA
-@pytest.mark.parametrize("train", (False, True))
-def test_parse_resnet18(turn_off_tf32_and_set_seed, train: bool):
+def test_parse_resnet18(executor, device, dtype, turn_off_tf32_and_set_seed, train: bool):
```

Comment:
It seems strange that the discrepancy is back here. Do you know if pytest is sensitive to the order here (i.e. does the fixture need to come first)?

Reply:
I think it's the new discrepancy coming from the grad check